common/static/js/src/logger.js

```diff
@@ -0,0 +1,91 @@
+;(function() {
+    'use strict';
+    var Logger = (function() {
+        // listeners[event_type][element] -> list of callbacks
+        var listeners = {},
+            sendRequest, has;
+
+        sendRequest = function(data, options) {
+            var request = $.ajaxWithPrefix ? $.ajaxWithPrefix : $.ajax;
```

Comment:
It looks like it gets used as the basis of a couple of login urls, as well. It may be dead code at this point. I'd have to do some digging to know for sure.